api/kyverno/v1/image_verification_types.go

```diff
@@ -187,11 +187,20 @@ type StaticKeyAttestor struct {
 	// Reference to a Secret resource that contains a public key
 	Secret *SecretReference `json:"secret,omitempty" yaml:"secret,omitempty"`
 
-	// Rekor provides configuration for the Rekor transparency log service. If the value is nil,
-	// or an empty object is provided, the public instance of
-	// Rekor (https://rekor.sigstore.dev) is used.
+	// Rekor provides configuration for the Rekor transparency log service. If an empty object
+	// is provided the public instance of Rekor (https://rekor.sigstore.dev) is used.
+	// +kubebuilder:validation:Optional
+	Rekor *Rekor `json:"rekor,omitempty" yaml:"rekor,omitempty"`
+
+	// CTLog provides configuration for validation of SCTs.
+	// If the value is nil, default ctlog public key is used
 	// +kubebuilder:validation:Optional
-	Rekor *CTLog `json:"rekor,omitempty" yaml:"rekor,omitempty"`
```

Comment:
Is this change backward compatible?

Reply:
Nope, I moved `IgnoreSCT` to `ctlog:` (it was previously in `rekor:`). The reason being, we cannot add TUF and ctlog pubkeys to rekor as well as they are not related.

Reply:
> I was thinking about keeping IgnoreSCT in rekor and publishing a warning that states that it has been moved to ctlog

Removing the field is not going to be a breaking change ?

Reply:
@eddycharly Answered here: https://github.com/kyverno/kyverno/pull/8166#discussion_r1309686167